### ISSUE TITLE
Twitter

### ISSUE THREAD

**Issue by kick317** [junaid33/opensource.builders#163]
Twitter is an American microblogging and social networking service on which users post and interact with messages known as "tweets". Registered users can post, like, and retweet tweets, but unregistered users can only read them.

**Comment by alexanderadam** [comment]
Mastodon is currently listed as Facebook alternative but I would clearly rather see it as an Twitter alternative.

**Comment by kick317** [comment]
https://github.com/tschellenbach/Stream-Framework

**Comment by jcklpe** [comment]
I agree that Twitter would be a better comparison to Mastodon than Facebook. 

I'm currently going through a bunch of tickets, but if I have time I might be able to correct stuff, and issue a pull request to have that corrected.
